velox/dwio/parquet/tests/duckdb_reader/ParquetTpchTest.cpp

```diff
@@ -191,6 +191,11 @@ TEST_F(ParquetTpchTest, Q10) {
   assertQuery(10, 5, 40, std::move(sortingKeys));
 }
 
+TEST_F(ParquetTpchTest, Q12) {
+  std::vector<uint32_t> sortingKeys{0};
+  assertQuery(12, 3, 20, std::move(sortingKeys));
```

Comment:
I find these numbers hard to understand. I wonder if there is a way to make this code more readable.

Reply:
Thanks for the suggestion, I have added suitable class variables to help with the readability, is this fine?